autoleagueplay/replays.py

```diff
@@ -2,24 +2,71 @@
 from enum import Enum
 from pathlib import Path
 from typing import Dict, Any
+import os
+import RattletrapPython.rattletrap as rat
+import json
 
 import requests
 from rlbottraining.history.metric import Metric
+from autoleagueplay.paths import WorkingDir
 from watchdog.events import LoggingEventHandler
 from watchdog.observers import Observer
 
-
 class ReplayPreference(Enum):
     SAVE = 'save'  # save to the default replays directory
     CALCULATED_GG = 'calculated_gg'  # save locally and upload to https://calculated.gg/
     IGNORE_REPLAY = 'ignore'
+    ANONYMIZE_REPLAY = 'anonym'
 
 
 def upload_to_calculated_gg(replay_path: Path):
     with open(replay_path, 'rb') as f:
         response = requests.post('https://calculated.gg/api/upload', files={'replays': f})
         print(f'upload response to {replay_path.name}: {response}')
 
+def anonymize_replay(replay_path: Path, working_dir: WorkingDir):
+    replay_name = str(os.path.basename(str(replay_path)).split('.')[0])
+    anonym_replay_path = working_dir.replays / str(replay_name + '.replay')
```

Comment:
What about remaning the replay file to something reasonable? Like "Beast from the East vs Skybot"? This means you will have to call the `anonymize_play()` function from the `run_match()` function https://github.com/RLBot/AutoLeaguePlay/blob/master/autoleagueplay/run_matches.py#L43-L48 where your other match info is available. Calling it from there actually makes a lot more sense if you ask me. You would also avoid passing the `working_dir` to ReplayMonitor for instance.

Reply:
I wanted to do that. i can even rename the in-game name, but i couldnt from replay.py
I'll work on those changes